tools/rebasehelpers/commitchecker/commitchecker.go

```diff
@@ -36,8 +36,13 @@ func main() {
 
 	errs := []string{}
 	for _, validate := range AllValidators {
-		err := validate(nonbumpCommits)
-		if err != nil {
+		if err := validate(nonbumpCommits); err != nil {
+			errs = append(errs, err.Error())
+		}
+	}
+	if len(os.Getenv("GODEP_RESTORE")) > 0 {
```

Comment:
No, I mean the check for changes to a vendored package since a bump should be done using git history, not commit titles